src/index.ts

```diff
@@ -21,32 +21,49 @@ export class TestxServer {
   private schema: string;
   private server: Server;
   private serverUrl: string;
-  private dbConnection: Knex;
+  private backendContext: { [id: string]: any };
```

Comment:
If we can we shouldn't use `any` and in this case is easy to avoid

Also `[id: string]` is like saying that you can use any key you want, this should be used only if you don't know the value that will be stored, but in our case we know exactlly what is going to be stored in the context

Finally I think is better to just remove `backendContext` and put `typeDefs`, `resolvers` and `dbConnection` as direrect properties of the object, also look at the next comments because maybe we don't need to `typeDefs` and `resolvers`

Reply:
You're right, @ b1zzu, I only used that in `backend Content` because I didn't know if it was a good idea to use helper classes :see_no_evil:

And we really can stop storing typeDefs, resolvers, and dbConnection, we just have to store the generated express instance.

Thx for the suggestions <3

Reply:
Awesome, I thought it could work but I wasn't sure, storing only the express app will simplify our code, but maybe we should still store the db connection in order to clean the database

Reply:
Oh, it's true